src/posts/2020-02-05-text-resizing-in-ios-and-android.md

```diff
@@ -20,30 +20,36 @@ tags:
   - mobile
 ---
 
-When you're designing for mobile, it's important to remember that some people will need to change the size of the content
-to read it easily. There are so many ways to view content on mobile devices, from zooming in to orienting the screen differently, so a failure to design in a dynamic environment can lead to accessibility issues. There are different ways you can attend to this, whether you're writing an app for iOS or Android.
+When you're designing for mobile, it's important to remember that some people will need to change the size of the content to read it easily. 
+
+There are so many ways to view content on mobile devices, from zooming in to orienting the screen differently, so a failure to design in a dynamic environment can lead to accessibility issues. There are different ways you can attend to this, whether you're writing an app for iOS or Android.
 
 ## Considerations for iOS
 
-In iOS, users can only zoom in on specified areas. To allow them to resize the type, use [dynamic type](https://developer.apple.com/design/human-interface-guidelines/ios/visual-design/typography/#dynamic-type-sizes), which allows them to [resize the type](https://docs.google.com/spreadsheets/d/1B7UjuckBckAQtILeBCqoIUe3Ze3R8ow2KOQSM_JJI_w/edit#gid=0) in your app.
+In iOS, users can use [Full Screen Zoom or Window Zoom](https://support.apple.com/guide/iphone/zoom-iph3e2e367e/ios) to magnify parts of the screen. To allow them to resize the type, use [dynamic type](https://developer.apple.com/design/human-interface-guidelines/ios/visual-design/typography/#dynamic-type-sizes), which allows them to resize the type in your app.
 
-If you manually limit how large your text scales it may cause issues for people who need a bigger font. 
+If you manually limit how large your text scales it may cause issues for people who need a bigger font-- you want the text to scale dynamically instead of stopping at the point you delimit, because that size may still not be visible to all people.
 
-[Minimum font-size](https://uxdesign.cc/guide-for-designing-better-mobile-apps-typography-5796495ef86f) should be 17pt.
+[Minimum font-size](https://uxdesign.cc/guide-for-designing-better-mobile-apps-typography-5796495ef86f) for body text should be 17pt.
 
 ## Considerations for Android
+
 Again, users can only zoom in on specified areas, so you'll need to use [scale-independent pixels](https://www.pixel-ruler.net/android-scale-independent-pixel) to determine the size of your text.
 
 There isn't a tool like Dynamic Type in Android, so you'll need to scale the text at different sizes yourself. Material design [has a resource](https://material.io/design/typography/the-type-system.html#type-scale) that makes a good starting point.
```

Comment:
Maybe mention that it can be helpful to use an emulated version of your app to see how it better works in context?

Reply:
Committed a couple sentences to address this!